cogdl/oag/oagbert_metainfo.py

```diff
@@ -251,21 +254,236 @@ def add_span(token_type_id, token_ids, is_mask=False):
             num_spans,
         )
 
+    def encode_paper(
```

Comment:
The codes written here are a bit redundant. Could you make it simpler? Maybe you can generalize different types of entities.